### DISCUSSION TITLE
Modelling of multiple materials and multiple reactions

### DISCUSSION THREAD

**Discussion by MohamedS2022** [firemodels/fds#12825] [Q&A]
Hi FDS team,

How can I model and simulate a FDS case that has multiple objects and each object has a dedicated material and related reaction?
also, how can get the HRR curve for each material contribution and the product of combustion outputs for each material like the below snapshot?
![FDS Case](https://github.com/firemodels/fds/assets/164491127/bb86feac-74cb-4c71-addc-85df9e7370c2)

**Comment by mcgratta** [comment]
If you are working with cables, look at the sample cases beginning with the word "cable" in the folder Pyrolysis.

**Reply by MohamedS2022** [reply]
Dear Prof. @mcgratta 

How i can get the location of this folder, i'm searching for it, Could you provide me with the link or the directory to it?
I have a facade system that consists of multiple materials and each material has an HRR curve and different products of combustion, this is the case that I have and I need to model all materials and investigate the contribution of each of them and the total of all material contributions.

**Comment by mcgratta** [comment]
What do you mean "each material has an HRR"? If the facade consists of multiple materials, how can you know the HRR of the composite?

The samples folder is where FDS is installed on your computer.

**Reply by MohamedS2022** [reply]
I mean that each object has a material and a pre-described heat release rate (HRR) curve.

**Comment by mcgratta** [comment]
I do not understand your plot. Is this the result of an experiment? If so, how was the experiment conducted and how were the component HRRs obtained?

**Reply by MohamedS2022** [reply]
Dear Prof. @mcgratta,
I do modeling for a paper that describes the facade system and its materials properties as below snapshots. 
![FDS Case-1](https://github.com/firemodels/fds/assets/164491127/5d864147-11c3-42af-a561-63de51c996f9)
![FDS Case-2](https://github.com/firemodels/fds/assets/164491127/ff1b933f-f01e-420b-b800-8dd5fcff4bb7)
![FDS Case-3](https://github.com/firemodels/fds/assets/164491127/ad3f867b-4406-40cb-9d68-afe527078f22)

and I will follow the approach for another paper that has the below HRR curves.
![FDS Case-4](https://github.com/firemodels/fds/assets/164491127/164b4512-beaa-48f0-9d2b-db7af286c288)
I hope that I described my case clearly to you and I appreciate your guidance and support.

**Comment by mcgratta** [comment]
I think what you want to do is convert an HRR curve from a full-scale burn of a particular material into detailed kinetic properties so that you can then model a material that is a composite of multiple materials. To my knowledge, there is no accepted way to do this. I suspect most would do it using some kind of iterative process where you estimate the properties and try to model the full-scale tests. Once you have a good match, you use those properties in the composite.
